certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "summary": "Webhook Trigger",
+        "description": "When a operation is performed on an entity in PPM Express",
+        "operationId": "Webhook_Trigger",
+        "x-ms-trigger": "single",
+        "x-ms-trigger-hint": "To see it work, perform any specified action in PPM Express",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "eventTypes": {
+                  "x-ms-summary": "Event Types",
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "Task:Created",
+                      "Task:Updated",
+                      "Task:Deleted",
+                      "Idea:Created",
+                      "Idea:Updated",
+                      "Idea:Deleted",
+                      "Project:Created",
+                      "Project:Updated",
+                      "Project:Deleted",
+                      "KeyDate:Created",
+                      "KeyDate:Updated",
+                      "KeyDate:Deleted"
+                    ],
+                    "title": "",
+                    "x-ms-visibility": "important"
+                  },
+                  "description": "eventTypes"
+                },
+                "headers": {
+                  "x-ms-summary": "Headers",
+                  "x-ms-visibility": "advanced",
+                  "type": "array",
+                  "items": {
+                    "description": "add headers",
+                    "title": "headers",
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "headers name",
+                        "title": "name",
+                        "default": "custom_header"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "headers value",
+                        "title": "value",
+                        "default": "custom_value"
+                      }
+                    }
+                  },
+                  "description": "headers_array,",
+                  "default": []
+                },
+                "httpVerb": {
+                  "type": "string",
+                  "description": "httpVerb",
+                  "title": "",
+                  "default": "POST",
+                  "x-ms-visibility": "internal"
+                },
+                "url": {
+                  "type": "string",
+                  "description": "url",
+                  "title": "",

```

Comment:
@vsorokin-ppmexpress already mentioned empty properties above, please check for empty title properties.